components/script_traits/Cargo.toml

```diff
@@ -7,18 +7,28 @@ authors = ["The Servo Project Developers"]
 name = "script_traits"
 path = "lib.rs"
 
+[dependencies.canvas_traits]
+path = "../canvas_traits"
+
 [dependencies.msg]
 path = "../msg"
 
 [dependencies.net_traits]
 path = "../net_traits"
 
+[dependencies.offscreen_gl_context]
+git = "https://github.com/ecoal95/rust-offscreen-rendering-context"
+features = ["texture_surface"]
+
 [dependencies.plugins]
 path = "../plugins"
 
 [dependencies.profile_traits]
 path = "../profile_traits"
 
+[dependencies.style_traits]
+path = "../style_traits"
+
```

Comment:
`ScriptMsg` references `canvas_traits::CanvasMsg`, `offscreen_gl_context::GLContextAttributes` and `style_traits::viewport::ViewportConstraints`. Those dependencies were removed from `msg` crate.